front/src/modules/rollingStock/components/RollingStockSelector/SearchRollingStock.tsx

```diff
@@ -149,6 +153,19 @@ const SearchRollingStock = ({
     setFilteredRollingStockList(newFilteredRollingStock);
   }
 
+  const resetFilters = () => {
+    if (mustResetFilters) {
```

Comment:
this test must be made in the useEffect hook, because, maybe in further iteration, there will be a button handled by this component to reset all the filter.

`mustResetFilters` come from the props it's why you should test it in the hook